app/controllers/helpers.rb

```diff
@@ -0,0 +1,55 @@
+module Helpers
+	def self.available
+		true
+	end
+
```

Comment:
The indentation here is way off and should be corrected. Take pride in the aesthetics of your code. 2 spaces, no tabs.